packages/forge2d/example/web/circle_stress.dart

```diff
@@ -128,13 +128,14 @@ class CircleStress extends Demo {
       final bodyDef = BodyDef();
       final groundBody = world.createBody(bodyDef);
 
-      final rjd = RevoluteJointDef()
+      final revoluteJointDef = RevoluteJointDef()
         ..initialize(body, groundBody, body.position)
         ..motorSpeed = -pi
         ..maxMotorTorque = 1000000.0
         ..enableMotor = true;
 
-      world.createJoint<RevoluteJoint>(rjd);
+      final revoluteJoint = RevoluteJoint(revoluteJointDef);
```

Comment:
@spydon what do we think of including the `addJoint` method in `Joint`? 

So we can do something like `RevoluteJoint(revoluteJointDef)..addTo(world)`.

Note: Open to naming suggestions. For example, wdyt of `RevoluteJoint(revoluteJointDef)..add(world)`?

Reply:
I like the current way that you have done it with `world.add(revoluteJoint)`, since joints are such a core concept of `Forge2D`, I don't think it matters that the world exposes that method.